### PR TITLE
Update edxapp default image to hawthorn.1-2.0.1

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

This image is a complete refactorization of the build process and
fixes issues on settings that were blocking some deployments like
mongodb port numbers not being formatted to integers.

## Proposal

Change image tag in variables.

:warning: depends on #137 :warning:

### CODE REVIEW

File: apps/edxapp/templates/lms/bc.yml.j2 [hunk 1]
```diff
@@ -39,8 +39,12 @@ spec:
       FROM {{ edxapp_image_name }}:{{ edxapp_image_tag }}
       USER 0
       {% if edxapp_theme_url is defined and edxapp_theme_url -%}
+      RUN apt-get update && \
+        apt-get install -y \
+        npm && \
+        rm -rf /var/lib/apt/lists/*
       COPY . /edx/app/edxapp/edx-platform/themes/custom-theme
-      RUN NO_PREREQ_INSTALL=1 paver update_assets --settings={{ edxapp_build_settings }} --skip-collect
+      RUN PATH=${PWD}/node_modules/.bin:${PATH} NO_PREREQ_INSTALL=1 paver update_assets --settings={{ edxapp_build_settings }} --skip-collect
```
Comment (sampaccoud) [inline apps/edxapp/templates/lms/bc.yml.j2]: @jmaupetit  this can be removed now?

File: apps/edxapp/vars/all/main.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ edxapp_lms_host: "lms.{{ project_name}}.{{ domain_name }}"
 
 # -- edxapp (cms/lms)
 edxapp_image_name: "fundocker/edxapp"
-edxapp_image_tag: "hawthorn.1-1.0.6"
+edxapp_image_tag: "hawthorn.1-2.0.0"
```
Comment (sampaccoud) [inline apps/edxapp/vars/all/main.yml]: upgrade to 2.0.1?

Comment (jmaupetit) [reply apps/edxapp/vars/all/main.yml]: Yes, we need to make a new edxapp release, upgrade to 2.0.1 and remove the PATH (your previous comment).

File: apps/edxapp/templates/lms/bc.yml.j2 (hunk 1 above)
Comment (jmaupetit) [reply apps/edxapp/templates/lms/bc.yml.j2]: Yes, once we've upgraded to `hawthorn.1-2.0.1`

**Comment by sampaccoud** [comment]
I can't approve it because I opened it but it definitely looks GTM!

**Comment by jmaupetit** [comment]
It's odd (strange) but I can :wink:

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**